docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-merge-pr.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: git-merge-pr
+description: Merges a specified open pull request.
+---
+
+# `git-merge-pr`
+
+`git-merge-pr` merges a specified open pull request. This step commonly follows a
+[`git-open-pr` step](git-open-pr.md) or can be used independently to merge existing
+pull requests.
+
+## Configuration
+
+| Name | Type | Required | Description |
+|------|------|----------|-------------|
+| `repoURL` | `string` | Y | The URL of a remote Git repository. |
+| `provider` | `string` | N | The name of the Git provider to use. Currently `azure`, `bitbucket`, `gitea`, `github`, and `gitlab` are supported. Kargo will try to infer the provider if it is not explicitly specified. |
+| `insecureSkipTLSVerify` | `boolean` | N | Indicates whether to bypass TLS certificate verification when interfacing with the Git provider. Setting this to `true` is highly discouraged in production. |
+| `prNumber` | `integer` | Y | The pull request number to merge. |
+| `wait` | `boolean` | N | If `true`, the step will return a running status instead of failing when the PR is not yet mergeable. The merge will be retried on the next reconciliation until it succeeds or times out. Default is `false`. Note: The `wait` option is unreliable for repositories hosted on Bitbucket due to API limitations. |
```

Comment:
```suggestion
| `wait` | `boolean` | N | If `true`, the step will return a running status instead of failing when the PR is not yet mergeable. The merge will be retried on the next reconciliation until it succeeds or times out. Default is `false`. Note: The `wait` option is unreliable for repositories hosted by Bitbucket due to API limitations. |
```

I would also link to docs about timeouts here.